free-programming-books-pl.md

```diff
@@ -84,6 +101,13 @@
 * [Haskell](https://pl.wikibooks.org/wiki/Haskell)
 
 
+### GameDev
+*Linki związane z szeroko pojętym tworzeniem gier.*
+
+* [tworzenie platformówek](http://informatyka.wroc.pl/gry) - bardzo fajne poradniki z tworzenia gier 2D
```

Comment:
This is a section of a website. I'm hesitant to add it because then why not add the other sections. It could be added as a website if it was deemed an important resource for Polish programmers.